### PR TITLE
[MIG] sale_product_set_packaging_qty: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**
Requires module:
- `sale_product_set` migrated in https://github.com/OCA/sale-workflow/pull/1399

### CODE REVIEW

File: sale_product_set_packaging_qty/models/product_set.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            )
+        else:
+            self.update({"product_packaging_qty": 0})
+
+    def prepare_sale_order_line_values(self, order, quantity, max_sequence=0):
+        res = super().prepare_sale_order_line_values(
+            order, quantity, max_sequence=max_sequence
+        )
+        if self.product_packaging_id:
+            res["product_packaging"] = self.product_packaging_id.id
```
Comment (kevinkhao) [inline sale_product_set_packaging_qty/models/product_set.py:76]: mistyped field, field name is product_packaging_id

**Review by kevinkhao — CHANGES_REQUESTED** [review]
Functional test

File: sale_product_set_packaging_qty/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "summary": "Manage packaging and quantities on product set lines",
+    "version": "14.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Warehouse Management",
+    "website": "https://github.com/OCA/sale-workflow",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": ["sale_product_set"],
```
Comment (kevinkhao) [inline sale_product_set_packaging_qty/__manifest__.py]: I think there is a missing dependency on sale_stock. On trying to add a product set with packaging:
Screenshot from 2021-03-26 16-39-07

**Review by kevinkhao — CHANGES_REQUESTED** [review]
Functional review

File: sale_product_set_packaging_qty/__manifest__.py (hunk 2 above)
Comment (kevinkhao) [reply sale_product_set_packaging_qty/__manifest__.py]: Apart from that LGTM

Comment (simahawk) [reply sale_product_set_packaging_qty/__manifest__.py]: @phuctranerp ping

File: test-requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+odoo14-addon-sale-product-set @ git+https://github.com/trobz/sale-workflow@14.0-mig-sale_product_set#subdirectory=setup/sale_product_set
```
Comment (simahawk) [inline test-requirements.txt]: pls drop this

**Review by simahawk — CHANGES_REQUESTED** [review]

File: test-requirements.txt (hunk 3 above)
Comment (phuctranfxvn) [reply test-requirements.txt]: thanks, fixed

File: sale_product_set_packaging_qty/__manifest__.py (hunk 2 above)
Comment (phuctranfxvn) [reply sale_product_set_packaging_qty/__manifest__.py]: sorry for late reply, I missed the message. fixed in the latest update. Thanks

File: sale_product_set_packaging_qty/__manifest__.py [hunk 4]
```diff
@@ -3,14 +3,14 @@
...
+    "version": "14.0.1.0.0",
     "development_status": "Alpha",
     "category": "Warehouse Management",
     "website": "https://github.com/OCA/sale-workflow",
     "author": "Camptocamp, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "application": False,
     "installable": True,
-    "depends": ["sale_product_set"],
+    "depends": ["sale_product_set", "sale_stock"],
```
Comment (rousseldenis) [inline sale_product_set_packaging_qty/__manifest__.py:14]: Why adding this dependency ? product.packaging is defined in product module and I don't see any requirements in code.

Comment (rousseldenis) [reply sale_product_set_packaging_qty/__manifest__.py:14]: Oh, didn't see above comments

File: sale_product_set_packaging_qty/__manifest__.py (hunk 2 above)
Comment (rousseldenis) [reply sale_product_set_packaging_qty/__manifest__.py]: Will you fix 13.0 too ?

**Review by simahawk — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by simahawk** [comment]
FTR @phuctranerp will backport to v13

/ocabot merge nobump

**Pull request merged**